src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPipelinePublisher.java

```diff
@@ -203,7 +204,7 @@ private Result runPipeline(
                 projectName,
                 null,
                 null,
-                null,
+                BuildTriggerSource.CI,
```

Comment:
This might have been intentionally left off previously, but I'm not sure. Both `ElectricFlowAssociateBuildToRelease` and `ElectricFlowTriggerRelease` set `BuildTriggerSource.CI`, so I thought it would make sense to add it here too.